shell/packages/sandstorm-ui-topbar/topbar.js

```diff
@@ -45,6 +45,7 @@ Template.sandstormTopbarBlockReload.onDestroyed(function () {
 
 Template.sandstormTopbar.onCreated(function () {
   Template.instance().popupPosition = new ReactiveVar(undefined, _.isEqual);
+  // TODO: subscribe to the list of open grains for this session
```

Comment:
If this is still relevant, add parenthetical (e.g. `TODO(soon)`) and add more details.